subgraphs/uniswap-v3/src/mappings/core.ts

```diff
@@ -47,17 +47,17 @@ export function handleMint(event: MintEvent): void {
 
 // Handle a burn event emitted from a pool contract. Considered a withdraw into the given liquidity pool.
 export function handleBurn(event: BurnEvent): void {
-  createWithdraw(
-    event,
-    event.params.owner,
-    event.params.owner,
-    event.params.amount0,
-    event.params.amount1,
-    false
-  );
-  updateUsageMetrics(event, event.params.owner, UsageType.WITHDRAW);
-  updateFinancials(event);
-  updatePoolMetrics(event);
+  // createWithdraw(
+  //   event,
+  //   event.params.owner,
+  //   event.params.owner,
+  //   event.params.amount0,
+  //   event.params.amount1,
+  //   false
+  // );
+  // updateUsageMetrics(event, event.params.owner, UsageType.WITHDRAW);
+  // updateFinancials(event);
+  // updatePoolMetrics(event);
```

Comment:
Don't be afraid to delete. If anything it will be in the git history :D

Reply:
Deleted.